src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -605,6 +619,26 @@ class xclbin_full : public xclbin_impl
       : std::make_pair(nullptr, size_t(0));
   }
 
+  std::vector< std::pair<const char*, size_t> >
+  get_axlf_sections(axlf_section_kind kind) const override
+  {
+    auto result = m_axlf_sections.equal_range(kind);
+
+    int count = std::distance(result.first, result.second);
+
+    if (count > 0) {
+      std::vector< std::pair<const char*, size_t> > return_sections;
+
+      for (auto itr = result.first; itr != result.second; itr++)
+	return_sections.emplace_back(std::make_pair((*itr).second.data(), (*itr).second.size()));
```

Comment:
Use spaces instead of tabs
```suggestion
        return_sections.emplace_back(std::make_pair(itr->second.data(), itr->second.size()));
```
or perhaps even
```
        return_sections.emplace_back(itr->second.data(), itr->second.size());
```